ci/datamgr.py

```diff
@@ -393,37 +695,12 @@ def dask(**params):
 
 
 @cli.command()
-@click.option('-t', '--tables', multiple=True, default=TEST_TABLES)
-@click.option(
-    '-d',
-    '--data-directory',
-    default=DATA_DIR,
-    type=click.Path(
-        exists=True,
-        file_okay=False,
-        dir_okay=True,
-        writable=True,
-        readable=True,
-        path_type=Path,
-    ),
-)
-@click.option('-i', '--ignore-missing-dependency', is_flag=True, default=True)
-def datafusion(tables, data_directory, ignore_missing_dependency, **params):
-    try:
-        import pyarrow as pa  # noqa: F401
-        import pyarrow.parquet as pq  # noqa: F401
-    except ImportError:
-        msg = 'PyArrow dependency is missing'
-        if ignore_missing_dependency:
-            logger.warning('Ignored: %s', msg)
-            return 0
-        else:
-            raise click.ClickException(msg)
-
-    for table, df in read_tables(tables, data_directory):
-        arrow_table = pa.Table.from_pandas(df)
-        target_path = data_directory / f'{table}.parquet'
```

Comment:
Forgot to add parquet tests to the datafusion binding hence we haven't been using the generated parquet files.

Reply:
Yep. This can be done in a follow up.